huaweicloud/services/cts/resource_huaweicloud_cts_data_tracker.go

```diff
@@ -130,6 +130,45 @@ func buildCreateRequestBody(d *schema.ResourceData) *cts.CreateTrackerRequestBod
 	return &reqBody
 }
 
+func expandResourceTags(tagmap map[string]interface{}) *[]cts.Tags {
+	taglist := make([]cts.Tags, 0, len(tagmap))
+
+	for k, v := range tagmap {
+		taglist = append(taglist, cts.Tags{
+			Key:   utils.String(k),
+			Value: utils.String(v.(string)),
+		})
+	}
+
+	return &taglist
+}
```

Comment:
please ues utils.ExpandResourceTags()

Reply:
Cannot use this method because the types of tags are different